components/sidebar/sidebar_category/sidebar_category_menu/sidebar_category_menu.tsx

```diff
@@ -96,105 +112,167 @@ const SidebarCategoryMenu = (props: Props) => {
+                labels={(
+                    <FormattedMessage
+                        id='user.settings.sidebar.sortAlpha'
+                        defaultMessage='Alphabetically'
+                    />
+                )}
+                onClick={(event) => handleSortChannels(event, CategorySorting.Alphabetical)}
+            />
+            <Menu.Item
+                id={`sortByMostRecent-${props.category.id}`}
+                labels={(
+                    <FormattedMessage
+                        id='sidebar.sortedByRecencyLabel'
+                        defaultMessage='Recent Activity'
+                    />
+                )}
+                onClick={(event) => handleSortChannels(event, CategorySorting.Recency)}
+            />
+            <Menu.Item
+                id={`sortManual-${props.category.id}`}
+                labels={(
+                    <FormattedMessage
+                        id='sidebar.sortedManually'
+                        defaultMessage='Manually'
+                    />
+                )}
+                onClick={(event) => handleSortChannels(event, CategorySorting.Manual)}
+            />
+        </Menu.SubMenu>
+    );
 
-    function handleCreateCategory() {
+    function handleCreateCategory(event: MouseEvent<HTMLLIElement>) {
+        event.preventDefault();
         props.openModal({
             modalId: ModalIdentifiers.EDIT_CATEGORY,
             dialogType: EditCategoryModal,
         });
         trackEvent('ui', 'ui_sidebar_category_menu_createCategory');
     }
 
+    const createNewCategoryMenuItem = (
+        <Menu.Item
+            id={`create-${props.category.id}`}
+            onClick={handleCreateCategory}
+            leadingElement={<FolderPlusOutlineIcon size={18}/>}
+            labels={(
+                <FormattedMessage
+                    id='sidebar_left.sidebar_category_menu.createCategory'
+                    defaultMessage='Create New Category'
+                />
+            )}
+        />
+    );
+
+    function handleMenuToggle(isOpen: boolean) {
+        setIsMenuOpen(isOpen);
+    }
+
     return (
-        <SidebarMenu
-            id={`SidebarCategoryMenu-${props.category.id}`}
-            ariaLabel={formatMessage({id: 'sidebar_left.sidebar_category_menu.dropdownAriaLabel', defaultMessage: 'Category Menu'})}
-            buttonAriaLabel={formatMessage({id: 'sidebar_left.sidebar_category_menu.dropdownAriaLabel', defaultMessage: 'Category Menu'})}
-            isMenuOpen={props.isMenuOpen}
-            onOpenDirectionChange={handleOpenDirectionChange}
-            onToggleMenu={handleToggleMenu}
-            tooltipText={formatMessage({id: 'sidebar_left.sidebar_category_menu.editCategory', defaultMessage: 'Category options'})}
+        <div
+            className={classNames(
+                'SidebarMenu',
+                'MenuWrapper',
+                {menuOpen: isMenuOpen},
+                {'MenuWrapper--open': isMenuOpen},
```

Comment:
if the condition is the same you can just use one extra class on it
```suggestion
                {menuOpen: isMenuOpen},
```

Reply:
changed